docs/messaging-implementation.md

```diff
@@ -22,6 +22,7 @@ point uportal-app-framework to your desired feed.
             "expireDate": "2017-08-02",
             "featureImageUrl": null,
             "priority": "high",
+            "recurrance": true,
             "dismissible": false,
```

Comment:
Consider making the `dismissible` property a String with several meaningful values ("false", "session", "true").  Recurrence feels more like a nuance of `dismissible` than a separate property? What would it mean for a notification to be non-dismissible but have recurrence set to `true`?

It may be possible to do this in a backwards-compatible way such that plain old `true` and `false` for `dismissible` continue to work, but even if it's not, that's probably ok because `dismissible` is called out as `experimental`. I don't think we have to semantically version experimental aspects of APIs.